packages/core/useColorMode/index.ts

```diff
@@ -60,6 +60,15 @@ export interface UseColorModeOptions<T extends string = BasicColorSchema> extend
    * @default localStorage
    */
   storage?: StorageLike
+
+  /**
+   * Omit `auto` mode from state
+   *
+   * When set to `true`, preferred mode is translated into `light` or `dark`
+   *
+   * @default true
+   */
+  omitAuto?: boolean
```

Comment:
Instead of "omit" and defaulting to true, I think it's better to have it as opt-in, like "emitAuto: false"

Reply:
Makes sense, feels more natural 👍 I pushed another commit to change it.